docker_enforcer.py

```diff
@@ -44,24 +45,26 @@ def setup_logging():
     if not (config.run_start_events or config.run_periodic):
         raise ValueError("Either RUN_START_EVENTS or RUN_PERIODIC must be set to True")
 
-    if config.run_start_events:
-        start_events = Observable.from_iterable(docker_helper.get_start_events_observable()) \
+    run_events = config.run_start_events or config.run_update_events or config.run_rename_events
+    if run_events:
+        events = Observable.from_iterable(docker_helper.get_events_observable()) \
             .observe_on(scheduler=NewThreadScheduler()) \
+            .where(lambda e: is_configured_event(e)) \
             .map(lambda e: e['id']) \
-            .map(lambda cid: docker_helper.check_container(cid))
+            .map(lambda cid: docker_helper.check_container(cid, True))
```

Comment:
@piontec Hi Lukasz,

This flag is used only for single check. Periodical will still use cache. This is mainly for update/rename events. If we do not remove container params from the cache then enforcer will compare rules with old container params.